auth0/src/main/java/com/auth0/android/request/Request.java

```diff
@@ -52,4 +54,33 @@
      */
     @NonNull
     T execute() throws Auth0Exception;
+
+    /**
+     * Add parameters to the request as a Map of Object with the keys as String
+     *
+     * @param parameters to send with the request
+     * @return itself
+     */
+    @NonNull
+    Request<T, U> addParameters(@NonNull Map<String, Object> parameters);
```

Comment:
how useful is keeping this method vs iteratively calling `addParameter(name, value)`? Maybe we should consider removing it today and adding it back later if we find a use case.

Reply:
Discussed - given that it's already present, and does have some convenience, we're going to keep it.